sdcm/nemesis.py

```diff
@@ -1115,6 +1115,41 @@ def disrupt_network_start_stop_interface(self):  # pylint: disable=invalid-name
         finally:
             self.target_node.remoter.run("sudo /sbin/ifup eth1")
 
+    def disrupt_streaming_error(self):
```

Comment:
I would like to have more streaming scenarios, not only decommission, but also rebuild and maybe add node.
It's ok to have them in separate PRs, but the naming should be change in order to differentiate them later.

Reply:
OK, I will add multiple nemesis with different streaming error

Reply:
For this PR you can just handle the naming to support later nemesis in follow up PRs.